src/template-parameters.json

```diff
@@ -268,7 +287,7 @@
       {
         "displayName": "Search Insights Source URL",
         "name": "searchInsightsSource",
-        "defaultValue": "https://cdn.jsdelivr.net/npm/search-insights@1.3.1",
+        "defaultValue": "https://cdn.jsdelivr.net/npm/search-insights@1.8.0",
```

Comment:
Did you make sure that the GTM integration works as expected with newer versions?